website/docs/r/compute_url_map.html.markdown

```diff
@@ -108,50 +106,44 @@ The following arguments are supported:
 
 * `description` - (Optional) A brief description of this resource.
 
-* `host_rule` - (Optional) A list of host rules. See below for configuration
-    options.
+* `host_rule` - (Optional) A list of host rules. Multiple blocks of this type are permitted. Structure is documented below.
 
-* `path_matcher` - (Optional) A list of paths to match. See below for
-    configuration options.
+* `path_matcher` - (Optional) A list of paths to match. Structure is documented below.
 
 * `project` - (Optional) The project in which the resource belongs. If it
     is not provided, the provider project is used.
 
-* `test` - (Optional) The test to perform. See below for configuration options.
+* `test` - (Optional) The test to perform.  Multiple blocks of this type are permitted. Structure is documented below.
 
-The `host_rule` block supports: (This block can be defined multiple times).
+The `host_rule` block supports:
 
-* `hosts` (Required) - A list of hosts to match against. See the documentation
-    for formatting each host
-    [here](https://cloud.google.com/compute/docs/reference/latest/urlMaps#hostRules.hosts)
+* `hosts` (Required) - The list of [host patterns](https://cloud.google.com/compute/docs/reference/latest/urlMaps#hostRules.hosts)
+ to match.
 
 * `description` - (Optional) An optional description of the host rule.
 
-* `path_matcher` - (Required) The name of the `path_matcher` (defined below)
-    to apply this host rule to.
+* `path_matcher` - (Required) The name of the `path_matcher` to apply this host rule to.
 
-The `path_matcher` block supports: (This block can be defined multiple times)
+The `path_matcher` block supports:
 
-* `default_service` - (Required) The URL for the backend service or backend bucket to use if none
-    of the given paths match. See the documentation for formatting the service/bucket
-    URL [here](https://cloud.google.com/compute/docs/reference/latest/urlMaps#pathMatcher.defaultService)
+* `name` - (Required) The name of the `path_matcher` resource.
 
-* `name` - (Required) The name of the `path_matcher` resource. Used by the
-    `host_rule` block above.
+* `default_service` - (Required) The URL or [valid partial url](https://cloud.google.com/compute/docs/reference/latest/urlMaps#pathMatcher.defaultService)
```

Comment:
If a valid partial url is set, will it return that same partial url on read? my gut says no, in which case we should request the full url to avoid perpetual diffs (or needing to set it to computed).